services/mailer.js

```diff
@@ -87,7 +89,9 @@ const mailer = module.exports = {
     }
 
     // Prefix the subject with `[Talk]`.
-    subject = `[Talk] ${subject}`;
+    subject = `[Talk] {t(${subject})}`;
```

Comment:
We won't support a dynamic use of `t` in the future. The subject should be translated before passing it to this method. See my comment above.